packages/nodes-base/nodes/Mattermost/v1/actions/router.ts

```diff
@@ -0,0 +1,53 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	INodeExecutionData, NodeApiError,
+} from 'n8n-workflow';
+
+import * as channel from './channel';
+import * as message from './message';
+import * as reaction from './reaction';
+import * as user from './user';
+
+export async function router(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
+	const items = this.getInputData();
+	const operationResult: INodeExecutionData[] = [];
```

Comment:
Subjective, but `operationResult` is quite wordy for a variable to be used so often. The original was `returnData`, which has `data` in the name, so probably best avoided as a var name. Maybe simply `output`? Thinking of future contributors and newcomers to the codebase.